www/base/src/components/ForceBuildModal/Fields/FieldChoiceString.tsx

```diff
@@ -22,29 +22,47 @@ import {observer} from "mobx-react";
 import {FieldBase} from "./FieldBase";
 import {Form} from "react-bootstrap";
 import {Tooltip} from 'react-tooltip'
+import Select, { ActionMeta, PropsValue, SingleValue, MultiValue } from 'react-select';
+import CreatableSelect from 'react-select/creatable';
 
 type FieldChoiceStringProps = {
   field: ForceSchedulerFieldChoiceString;
   fieldsState: ForceBuildModalFieldsState;
 }
 
+interface SelectOption
+{
+  readonly value: string,
+  readonly label: string,
+}
+
+const ValueToSelectOption = (value: string): SelectOption => { return {value: value, label: value} };
+
 export const FieldChoiceString = observer(({field, fieldsState}: FieldChoiceStringProps) => {
   const state = fieldsState.fields.get(field.fullName)!;
+  if (field.multiple && !Array.isArray(state.value)) {
+    fieldsState.setValue(field.fullName, state.value ? [state.value] : []);
+  }
+
+  const onChange = (
+    newValue: PropsValue<SelectOption>,
+    _actionMeta: ActionMeta<SelectOption>
+  ) => {
+    fieldsState.setValue(
+      field.fullName,
+      (
+        field.multiple ?
```

Comment:
I think something like:

```
condition ? b
                  : c;
```

would be more readable in this case. I was genuinely confused for a bit.